.github/workflows/tests.yaml

```diff
@@ -11,25 +11,20 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                php: ['7.3', '7.4']
+                php: ['7.3', '7.4', '8.0']
 
         name: PHP ${{ matrix.php }} tests
         steps:
             -   uses: actions/checkout@v2.0.0
-            -   id: composer-cache
-                run: echo "::set-output name=dir::$(composer config cache-files-dir)"
-            -   uses: actions/cache@v1
-                with:
-                    path: ${{ steps.composer-cache.outputs.dir }}
-                    key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-                    restore-keys: |
-                        ${{ runner.os }}-composer-
+
             -   uses: shivammathur/setup-php@v2
                 with:
-                    php-version: 7.3
```

Comment:
It seems tests only run on PHP 7.3.
That explain why I had so many errors on PHP 8.0 :)

Reply:
@TomasVotruba Oh no, stupid me. Thanks for fixing.